### PR TITLE
some PW fixes

### PR DESCRIPTION
**Pull request by kitala1**
PW now spawns the proper lamps, doesn't crash server upon trading Pandemonium Key, and readded 2 lamp IDs the database that were deleted for some reason. PW is not mostly funtional and is ripe for murdering.

### CODE REVIEW

**Comment by kitala1** [comment]
I'm awful at git, feel free to leave some feedback <_<

I deleted the changes from fix '9a1e79e', but it's still listed for some reason.... lawl

**Comment by teschnei** [comment]
replace all your tabs with 4 spaces each

rename your commit messages to include the labels (it's in the readme on the front page)

**Comment by kitala1** [comment]
Sorry about that, you probably have to say that 100 times a day... will fix

**Comment by teschnei** [comment]
no need to close it, you can just push new commits fixing it to your remote branch and they'll show up here instantly

File: scripts/zones/Aydeewa_Subterrane/mobs/Pandemonium_Warden.lua [hunk 1]
```diff
@@ -34,6 +29,12 @@ end;
 -----------------------------------
 
 function onMobEngaged(mob,target)
+	-- Two hours to forced depop
```
Comment (TeoTwawki) [inline scripts/zones/Aydeewa_Subterrane/mobs/Pandemonium_Warden.lua]: Don't do that. If you do that they can reset teh timer simply by deagro'ing it for a splti secodn and reclaiming before it depops.

**Comment by TeoTwawki** [comment]
@kitala1  What was it crashing on exactly.

**Comment by kitala1** [comment]
That code is actually already in the DSP git. Before a few days ago that section of code was commented out like this:

-- Uncomment if you're testing with @spawnmob, otherwise you'll get all kinds of nil errors and no pets, while onMobFight doesn't run.
-- Leave commented in most situations, otherwise players can reset their despawn timer.
--[[
	PWardenDespawnTime = os.time(t) + 7200; -- Two hours to forcible depop, if PWarden is engaged.
 	PWardenAstralFlows = 0;

	for i = 17056170, 17056177, 1 do 
		SpawnMob(i,180):updateEnmity(target);
	end
]]--

**Comment by kitala1** [comment]
sorry for that sloppy comment above, im not sure how to add code to these. PW was crashing the server when you traded him the pandemonium key on my server, all i did was change the order in qm2.lua

**Comment by TeoTwawki** [comment]
The timer
the time doesn't go in onEngaged

**Comment by TeoTwawki** [comment]
Not even talkin about the pets

**Comment by TeoTwawki** [comment]
or you mean that was crashing? I thought you meant the crash was in the trade script, sorry. Multi tasking a lot

**Comment by kitala1** [comment]
SpawnMob(17056168,180):updateClaim(player); -- Pandemonium Warden
player:tradeComplete(); 

this was crashing the server, it was fixed it by switching these two lines.

**Comment by teschnei** [comment]
interesting that you have a commit from 2014 lingering in your branch..

**Comment by TeoTwawki** [comment]
@kitala1 are you sure it was those 2 liens causing it, and not the mistake I moved into onSpawn for the monster the trade was spawning? 

A server I help out a ton, has had PW working and when they pull requested they left a bunch of custom stuff in by mistake. Their guy had rl work, so I went about cleaning it out. When i did I was asked to change a bunch of other stuff (like those for loops - demo wanted just one loop rather than copying the spawn mobs commands into each "elseif" block).

**Comment by kitala1** [comment]
I'm not sure how to remove that... it was still in my fork when i synced it to my pc, so i just overwrote the files that were conflicting manually <_<;

**Comment by TeoTwawki** [comment]
rebase should get rid of it

**Comment by teschnei** [comment]
never write your pull request on master, make a branch for each one you do

**Comment by kitala1** [comment]
I think I know what you're saying... I'm very new to git still. Should I resubmit this pull or something? Sorry again for being ignorant

**Comment by TeoTwawki** [comment]
@kitala1 You able to join us on IRC so I don't keep spamming devs email inbox? ```#darkstarproject``` on rizon.

**Comment by kitala1** [comment]
on there now

**Comment by kitala1** [comment]
Thanks for your help Teo~

**Comment by TeoTwawki** [comment]
Now you just need to rebase to fix the commit history :thumbsup:

**Comment by xdemolish** [comment]
commit labels please (see https://github.com/DarkstarProject/darkstar/blob/master/README.md)

**Comment by TeoTwawki** [comment]
Looks all legit now n stuff.

@kitala1 tip: they don't get messages telling them when we adjust things in the pr, so we have to post a  comment letting them know.

**Comment by kitala1** [comment]
understood! :)

**Pull request merged**